### PR TITLE
16 eth/btc websocket bugs and improves

### LINKED ISSUE #16: ETH/BTC Websocket bugs and improves
- [x] I have read the [Contributing](https://github.com/swapper-org/NodeChain/blob/master/CONTRIBUTING.md) file of the documentation.
- [x] I have searched the [issues](https://github.com/swapper-org/NodeChain/issues) (including closed ones) and believe that this is not a duplicate.

- **Link to your source / [Gist](https://gist.github.com/)**: 

[Unify ETH/BTC websocket responses](https://gist.github.com/swapper-phoenix/74df30367ee9b1d53c66a9e5e93b629e)
[ConnectionResetError: Cannot write to closing transport](https://gist.github.com/swapper-phoenix/296355676d4dff48be18ca4715ba32a9)
[TypeError: 'NoneType' object is not subcriptable in `getBlockByNumber`](https://gist.github.com/swapper-phoenix/78f4ef7e3181c953b856654945fe399f)

## Issue

- [x]  ETH and BTC response formats are different, we should unify them.
- [x] Add `address` field to `subscribeAddressBalance`,  `getAddressBalance` and `getAddressesBalance` in BTC/BCH/ETH
- [ ] Bitcoin WebSocket accumulates messages when you subscribe and unsubscribe from the same address. Instead of sending you a single message, it sends you as many messages as you have subscribed in the past (I suspect because the subscription info is persisted in Electrum and not only in the Connector).
- [ ] After a while of inactivity in the WebSocket and after connecting and disconnecting several times, when the WebSocket should send a message this [error trace](https://gist.github.com/swapper-phoenix/296355676d4dff48be18ca4715ba32a9) appears in the connector logs.
- [x] TypeError: 'NoneType' object is not subcriptable in `getBlockByNumber`

### PR DESCRIPTION
**Pull request by 0xp3gasus**
# Description

Fixes #16 

## Type of change

- [x] **Bug fix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **This change requires a documentation update**

# How Has This Been Tested?

- Tests already created run correctly and new test (`testAdressBalanceWS`) for ETH to validate the correct operation of WS Address Balance

Test passed

**Test Configuration**:
Operating system (output of `sw_vers`): macOS 12.0.1

# Related PR or Docs PR

Docs PR related # 
Other PR related # 

# Good practices to consider

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,23 +13,35 @@ jobs:
...
-          name: Deploy Development ETH Connector
+          name: Deploy ETH development nodes
           command: |
             docker-compose -f Connector/tests/docker-compose/eth.yml -p eth_development_api up --build -d
-      - run: 
+      - run:
+          name: Add ethereumgo to /etc/hosts
+          command: |
+            echo "127.0.0.1 ethereumgo" | sudo tee -a /etc/hosts
+            cat /etc/hosts
```
Comment (phoenix-web3) [inline .circleci/config.yml:35]: Is this line showed as logs?

File: .circleci/config.yml [hunk 2]
```diff
@@ -43,31 +55,45 @@ jobs:
...
-          name: Deploy Development BTC Connector
+          name: Deploy BTC development nodes
           command: |
             docker-compose -f Connector/tests/docker-compose/btc.yml -p btc_development_api up --build -d
+      - run:
+          name: Add electrum and bitcoincore to /etc/hosts
+          command: |
+            echo "127.0.0.1 electrum" | sudo tee -a /etc/hosts
+            echo "127.0.0.1 bitcoincore" | sudo tee -a /etc/hosts
+            cat /etc/hosts
```
Comment (phoenix-web3) [inline .circleci/config.yml:79]: Same question

File: Connector/eth/utils.py [hunk 3]
```diff
@@ -45,71 +37,16 @@ def getWSResponseMethodSchema(name):
...
-    logger.printInfo("Sending balance to subscribers")
-
-    for client in addressClients:
-
-        await client.websocket.send_str(
-            json.dumps(
-                balance
-            )
-        )
+def isAdressInBlock(address, block):
```
Comment (phoenix-web3) [inline Connector/eth/utils.py]: Typo: Address*

**Review by phoenix-web3 — COMMENTED** [review]
Review this changes @swapper-pegasus

**Comment by phoenix-web3** [comment]
You might need to rebase because of #43 @swapper-pegasus

File: .circleci/config.yml (hunk 1 above)
Comment (0xp3gasus) [reply .circleci/config.yml:35]: Line 34 is mandatory, line 35 just for logging

File: .circleci/config.yml (hunk 2 above)
Comment (0xp3gasus) [reply .circleci/config.yml:79]: Line 77 and 78 are mandatory, line 79 just for logging

File: Connector/eth/utils.py (hunk 3 above)
Comment (0xp3gasus) [reply Connector/eth/utils.py]: Resolved

**Review by phoenix-web3 — APPROVED** [review]

**Pull request merged**